### PR TITLE
Bug/fix timer issue

### LINKED ISSUE #139: Fix Timer Issue
## Description

Remove lee's old timers as it does not account for daylight savings time

### PR DESCRIPTION
**Pull request by hhamud**
---
name: Pull Request
about: Create a pull request for this project

---

## Description
* closes #139 
* replaces functions across components to enforce consistency

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/utils/date.ts [hunk 1]
```diff
@@ -16,14 +16,14 @@ export const convertUtcTimestampToLocal = (utcTimestamp: number) => {
   return dateUTC.unix()
 }
 
-export const convertTimestampWithMoment = (utcTimestamp: number) => {
+export const faketimer = (utcTimestamp: number) => {
```
Comment (adamazad) [inline src/utils/date.ts:19]: `camelCase`

File: src/utils/date.ts [hunk 2]
```diff
@@ -16,14 +16,14 @@ export const convertUtcTimestampToLocal = (utcTimestamp: number) => {
...
   return dayjs
     .utc(utcTimestamp * 1000)
     .tz(dayjs.tz.guess())
     .format('MMM D, hh:mm z')
 }
 
-export const calculateTimeDifference = (utcTimestamp: number) => {
+export const calculatefaketimer = (utcTimestamp: number) => {
```
Comment (adamazad) [inline src/utils/date.ts:26]: `camelCase`

File: src/views/Sale/components/Timer/index.tsx [hunk 3]
```diff
@@ -1,6 +1,7 @@
 // External
 import React, { useEffect, useState } from 'react'
 import dayjs from 'dayjs'
+import moment from 'moment-timezone'
```
Comment (adamazad) [inline src/views/Sale/components/Timer/index.tsx:4]: Isn't dayjs UTC plugin sufficient? momentjs is deprecated and has a huge bundle size

File: src/views/Sale/FixedPrice.tsx [hunk 4]
```diff
@@ -194,26 +194,21 @@ export function FixedPriceSaleView() {
...
+                        description={timeFrame(sale.startDate)}
                         textAlign="right"
                       />
                     )}
                     {isSaleOpen(sale) && (
                       <HeaderItem
                         isMobile
                         title="Ends In"
-                        description={calculateTimeDifference(sale.endDate)}
+                        description={secondsTohms(sale.endDate)}
```
Comment (adamazad) [inline src/views/Sale/FixedPrice.tsx:211]: `secondsToHms` maybe?

File: src/utils/date.ts (hunk 1 above)
Comment (hhamud) [reply src/utils/date.ts:19]: These functions are not used anymore. I will remove them

File: src/utils/date.ts (hunk 2 above)
Comment (hhamud) [reply src/utils/date.ts:26]: These functions are not used any. I will remove them

File: src/views/Sale/components/Timer/index.tsx (hunk 3 above)
Comment (hhamud) [reply src/views/Sale/components/Timer/index.tsx:4]: I've removed a library and have instead mapped the shorthand using my own function as Dayjs has deprecated this.

**Pull request closed without merging**